### PR TITLE
feat(QIcon/lang): sync QIcon implementation and lang with Q2

### PR DESCRIPTION
**Pull request by pdanpdan**

### CODE REVIEW

File: ui/lang/ar.js [hunk 1]
```diff
@@ -30,20 +30,18 @@ export default {
     noData: 'لا توجد بيانات',
     noResults: 'لا توجد نتائج',
     loading: 'جارٍ التحميل...',
-    selectedRecords: function (rows) {
-      return rows === 0
+    selectedRecords: rows => (
```
Comment (rstoenescu) [inline ui/lang/ar.js]: Arrow functions will break IE11. The regular function syntax is used on purpose for Qv1.

File: ui/src/plugins/Notify.js [hunk 2]
```diff
@@ -64,6 +64,14 @@ function logError (error, config) {
 const Notifications = {
   name: 'QNotifications',
 
+  // hide App from Vue devtools
+  devtools: { hide: true },
```
Comment (rstoenescu) [inline ui/src/plugins/Notify.js:68]: Isn't this a Vue 3 feature of devtools?
Why is this needed?

File: ui/build/script.build.javascript.js [hunk 3]
```diff
@@ -51,6 +51,7 @@ const builds = [
       },
       output: {
         file: resolve(`dist/quasar.common.js`),
+        exports: 'auto',
```
Comment (rstoenescu) [inline ui/build/script.build.javascript.js]: I'd rather we not go through the upgrade hell of dev deps and testing all of the flavours of Quasar. This adds no benefit to the end user but adds possible failure points for us.
So please downgrade back all deps.

**Review by rstoenescu — CHANGES_REQUESTED** [review]

File: ui/lang/ar.js (hunk 1 above)
Comment (pdanpdan) [reply ui/lang/ar.js]: Ok, will do, I forgot this is not transpiled

File: ui/src/plugins/Notify.js (hunk 2 above)
Comment (pdanpdan) [reply ui/src/plugins/Notify.js:68]: Vue devtools beta works with vue2, and it:
- shows the top level app
- borks on getting router for these plugins

File: ui/build/script.build.javascript.js (hunk 3 above)
Comment (pdanpdan) [reply ui/build/script.build.javascript.js]: Ok
